# State Msg to be given to GUI

string arm_state  # State of the arm
uint8 arm_controller_state # 0: ENABLED 1: DISABLED 2: CRITICAL

